packages/swc/README.md

```diff
@@ -0,0 +1,70 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-swc
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-swc
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-swc
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-swc
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-swc
+
+🍣 A Rollup plugin to transpile TypeScript/JavaScript with the speedy-web-compiler (swc).
+
+The plugin makes it possible to avoid the usage of `@rollup/plugin-babel` and `@rollup/plugin-typescript`.
+It is also blazingly fast 🔥 (between 20 - 70 times faster than Babel depending on how many cpu cores are used).
+
+## Requirements
+
+This plugin requires an [LTS](https://github.com/nodejs/Release) Node version (v10.0.0+) and Rollup v2.0+.
```

Comment:
I think this is meant to be Node 14+ and Rollup 3+

Reply:
Rollup v2 also works, which is also the case for most other plugins in this repository

Reply:
@tada5hi our baseline is rollup v3 for new plugins. I'll also add that to the contributing docs.

Reply:
I based that on @shellscape comment at https://github.com/rollup/plugins/pull/1412#pullrequestreview-1267973004

Reply:
Good Point 👍. I will change that asap.

Reply:
done